src/OpenLoco/Windows/Vehicle.cpp

```diff
@@ -1512,7 +1512,7 @@ namespace OpenLoco::Ui::Windows::Vehicle
                     str = StringIds::vehicle_details_total_power_and_weight;
                 }
                 Gfx::drawStringLeftClipped(*context, pos.x, pos.y, self->width - 6, Colour::black, str, &args);
-                pos.y += 10;
+                pos.y += 12;
```

Comment:
spacing of 12px is more consistent with other windows, and looks nicer as it doesn't overly cramp the text